### ISSUE TITLE
I don't understand logic in get_queryset() function

### ISSUE THREAD

**Issue by kerry407** [chukaibejih/smart_learn#12]
![Screenshot (299)](https://user-images.githubusercontent.com/63162639/228326591-4639ef83-b1b9-43a4-a0a9-386da8dfb806.png)
What does the highlighted function do? Because all users are active = True by default even superusers and staff accounts.

**Comment by kerry407** [comment]
Okay I think I see it now, you want to only return active users to people who use the app and all for only staffs

**Comment by chukaibejih** [comment]
Yes, it returns active users. In case in future where some account get deactivated or something that makes is_active = false.
